explorer/src/components/WalletSideKick/AccountPreferencesModal.tsx

```diff
@@ -36,6 +38,11 @@ export const AccountPreferencesModal: FC<ActionsModalProps> = ({ isOpen, prefere
             addresses.map((a) => a.address),
             'Address already exists in the address book',
           )
+          .test(
+            'is-valid-address',
+            'Invalid address (must be a valid Polkadot or Ethereum address)',
+            (value) => isAddress(value) || isEvmAddress(value),
```

Comment:
Are we intending to do EVM transactions in the sidekick?

Reply:
I was trying to keep the door open but I removed it